public/assets/css/header.css

```diff
@@ -0,0 +1,120 @@
+header{
+    display: flex;
+    align-items: center;
+    justify-content:space-between;
+    background-color: #facfcf;
```

Comment:
Could you set a global variable since you're using this color for multiples items